pkg/oauth/server/osinserver/registrystorage/storage.go

```diff
@@ -63,6 +63,10 @@ func (w *clientWrapper) GetUserData() interface{} {
 	return w.client
```

Comment:
we're already returning the client object here... we should be able to get the grant strategy from it